pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -466,6 +466,10 @@ func TestDesiredLoadBalancerService(t *testing.T) {
 				t.Errorf("local-with-fallback annotation check for test %q failed: %v", tc.description, err)
 			}
 		}
+		// Only make this assertion if we expect a service.
+		if tc.expect {
+			assert.Equal(t, "None", string(svc.Spec.SessionAffinity))
```

Comment:
Historical note: https://github.com/openshift/cluster-ingress-operator/pull/886/commits/a3bb33ec0e7fcae199024d38dbb113396130a4ef adds an `if !tc.expect || !haveSvc { return }` guard.  The `if tc.expect` needed to be added here as #886 wasn't backported to the release-4.13 branch.